.circleci/config.yml

```diff
@@ -38,7 +38,8 @@ jobs:
   # at https://hub.docker.com/r/circleci/node/.
 
   NodeJS 12:
-    executor: { name: oss/node, tag: '12' }
+    docker:
+    - image: cimg/node:12.22.1
     steps:
       - common_test_steps
       # We will save the results of this one particular invocation to use in
```

Comment:
Thoughts on if we should bump this to the 14 or 16 job?

Reply:
I had written out a longer comment but somehow lost it.  Let's wait and ticket this for follow-up in a few weeks? (Feel free to assign to me).  Basically, this `persist_to_workspace` thing propagates things to the Tarballs and Publishing steps: https://github.com/apollographql/CircleCI-Orbs/blob/main/src/oss-ci-cd-tooling/orb.yml#L319-L346

Reply:
still curious about this fwiw, but merging @abernix